terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -408,7 +340,24 @@ resource "kubernetes_deployment" "intake_batch" {
         container {
           name  = "facile-container"
           image = "${var.container_registry}/${var.facilitator_image}:${var.facilitator_version}"
-          args  = ["intake-batch-worker"]
+          args = [
+            "--pushgateway=${var.pushgateway}",
```

Comment:
`facilitator` doesn't use the Prometheus pushgateway. I think this has just been there since before `facilitator` ran as a persistent deployment pulling tasks from queues (in the long ago, the dark ages, just about a year ago). We should be able to just remove this argument. I notice that `facilitator/src/bin/facilitator.rs` still has this argument defined, which is a shame, because it doesn't do anything. I'd say we should remove that, too, but I'm worried it might break existing deployments if `clap` rejects unexpected arguments.

Reply:
I think it would almost certainly be OK removing the flag entirely, since it has only been specified as an environment variable in the past. The one wrinkle is the Google deployment, where I'm not sure if flags are specified as flags or environment variables.

For now I deprecate, once this change has gone out long enough I suggest we remove. Perhaps by then I'll have peeked into Google's configuration & will know how they specify flags.